src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheckTest.java

```diff
@@ -41,7 +41,7 @@ protected String getPackageLocation() {
     public void testGetRequiredTokens() {
         final AnnotationLocationCheck checkObj = new AnnotationLocationCheck();
         assertArrayEquals(
-            "AnnotationLocationCheck#getRequiredTockens should return empty array by default",
+            "AnnotationLocationCheck#getRequiredtokens should return empty array by default",
```

Comment:
Shouldn't `tokens` be capital `T`?
Almost all of these follow this pattern.

Reply:
done.